problem_builder/tests/integration/test_step_builder.py

```diff
@@ -624,9 +624,6 @@ def test_review_tips(self):
 
     @data(True, False)
     def test_conditional_messages(self, include_messages):
-        """
```

Comment:
Removed to see which one failed in the logs (nose shows docstrins over method names, and method names are mangled by ddt, so I couldn't run this test.